ci/scripts/python_test.sh

```diff
@@ -54,4 +54,4 @@ export PYARROW_TEST_ORC
 export PYARROW_TEST_PARQUET
 export PYARROW_TEST_S3
 
-pytest -r s -v ${PYTEST_ARGS} --pyargs pyarrow
+pytest -r P -v ${PYTEST_ARGS} --pyargs pyarrow
```

Comment:
Hmm, why the change? Should we undo it?

Reply:
yes, this was only temporary to be able to see the print output on CI. I'll revert it back.

Reply:
removed